packages/core/admin/admin/src/pages/MarketplacePage/components/SortFilter/index.js

```diff
@@ -1,28 +1,47 @@
 import React from 'react';
-import { SimpleMenu, MenuItem } from '@strapi/design-system/SimpleMenu';
+import { Select, Option } from '@strapi/design-system/Select';
 import { useIntl } from 'react-intl';
-import { NavLink } from 'react-router-dom';
+import { Typography } from '@strapi/design-system/Typography';
+import PropTypes from 'prop-types';
 import { getTrad } from '../../../../content-manager/utils';
 
-const SortFilter = () => {
+const SortFilter = ({ value, onChange }) => {
```

Comment:
I don't think Filter is the right word, because it doesn't actually filter out anything, it just reorders the results. Maybe SortSelect?